corehq/apps/es/filters.py

```diff
@@ -106,14 +92,14 @@ def doc_id(doc_id):
 def missing(field, exist=True, null=True):
```

Comment:
I just noticed this - these args are now ignored with this change.  At a glance, it looks like those aren't ever set to anything but `True`, so it should be safe to remove those kwargs.

Reply:
also, I was concerned this was a behavior change, since we always set `null_value` to True, which wasn't the default.
Relevant docs: [missing query](https://www.elastic.co/guide/en/elasticsearch/reference/2.2/query-dsl-missing-query.html#_literal_existence_literal_and_literal_null_value_literal_parameters), [exists query](https://www.elastic.co/guide/en/elasticsearch/reference/2.2/query-dsl-exists-query.html)
However I think that's actually okay, and it was actually the ES defaults that didn't make sense.

`exists()` matches `""`, and doesn't match `None` or `<missing>`

The old behavior for `missing()` matches <missing>, `None` (though not by default) and doesn't match `""`, which is the exact inverse of `exists()`, so there shouldn't be any change.

Reply:
Great note, Ethan. I have removed the no longer necessary kwargs